### DISCUSSION TITLE
Link beetween sections

### DISCUSSION THREAD

**Discussion by gilleschapron** [GooRoo/mkdocs-obsidian-bridge#12] [Q&A] [answered]
Hi,

Following our previous discussion, I have another problem, but not sure is related to your plugin.  

When I use wikilinks between folder is Obsidian, its fine in Obsidian, but the generated link is not.   

For example, I have written a wikilink in my `docs/fr/general/dst.md` file to my `docs/fr/tools/index.md` file.

But the generated html link is `general/dst/..%5Ctools%5Cindex.md` instead of simply `tools/`....

It's the same with markdown links syntax...

But with with autorefs plugin, when I use a cross-reference (https://mkdocstrings.github.io/usage/#cross-references), `[modules][modules]` it works.

Any idea to solve this case ?

**Comment by GooRoo** [comment]
Looks like a bug. I'll try to reproduce it on my side.

**Reply by GooRoo** [reply, marked as answer]
Okay, this should be fixed in #14 now (available since v1.0.4).
